server/app/routes/api/v1/va/ambassadors.js

```diff
@@ -526,6 +526,10 @@ async function claimTriplers(req, res) {
   const claimedTriplerCount = claims.length; // TODO: look up more idiomatic access
 
   let triplerLimit = ov_config.claim_tripler_limit;
+  const triplerLimitOverride = Number.parseInt(ambassador.get('claim_tripler_limit'));
+  if(Number.isSafeInteger(Number.parseInt(triplerLimitOverride))){
```

Comment:
looks like parseInt is getting called twice on this value, could you remove the second one?

Reply:
yup, done